dysnix/geth/values.yaml

```diff
@@ -1,126 +1,246 @@
 image:
-  repository: "ethereum/client-go"
-  ## AppVersion is used when not set
-  # tag: ""
+  repository: ethereum/client-go
   pullPolicy: IfNotPresent
+  tag: ""
 
 imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
 
-proxy:
-  enabled: false
-  http:
-    port: 9545
-  ws:
-    port: 9546
-  otpSecret: "<change-me>"
-  image:
-    pullPolicy: IfNotPresent
-    repository: dysnix/protected-proxy
-    tag: latest
-
-http:
-  port: 8545
-  api: "net,eth,personal,web3"
-  vhosts: "*"
-ws:
-  port: 8546
-  api: "net,eth,personal,web3"
-  origins: "*"
-authrpc:
-  enabled: false
-  port: 8551
-  # Use a utility like OpenSSL to create the token via command: openssl rand -hex 32 | tr -d "\n" > "jwt.hex".
```

Comment:
WDYT on keeping this comment in values.yaml ?

Reply:
Yup i think i just missed it. Fixed.